server/src/main/java/org/elasticsearch/cluster/SnapshotsInProgress.java

```diff
@@ -42,23 +42,78 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 
 /**
  * Meta data about snapshots that are currently executing
  */
 public class SnapshotsInProgress extends AbstractNamedDiffable<Custom> implements Custom {
```

Comment:
This diff is not readable, can you please move things back in the file. It would be much easier for the reviewers to see what exactly was changed and you won't be displayed in git history as the author for the lines that you have just re-shuffled.

Reply:
Sure we can shuffle that later :), reverted that part and fixed PR title.